assets/src/app/ato/Documents/Fedramp.tsx

```diff
@@ -28,6 +28,11 @@ class ATOFedRAMPs extends React.Component<any, FedRAMPsState> {
         }).map(function(p, idx) {
             return [
                 (<Text component="p" key={idx}>{p.name}</Text>),
+                /* (<TextContent key={idx+'t'}>
+                <center>
+                    <Text component="a" href={'/api/v1/components/' + p.key + '/fedramp/Tailored'}><FileWordIcon/> .docx</Text>
```

Comment:
`/fedramp/Tailored` is not provided by the backend.

What is the tailored supposed to do?